### ISSUE TITLE
rocksdb compilation fails with fatal error: 'stdarg.h' file not found

### ISSUE THREAD

**Issue by not-my-profile** [oxigraph/oxigraph#272]
I tried to run `cargo install oxigraph_server` but the installation failed with:

```
error: failed to run custom build command for `oxrocksdb-sys v0.3.7`

Caused by:
  process didn't exit successfully: `/tmp/cargo-installA1C1RJ/release/build/oxrocksdb-sys-7e1079602337d3f3/build-script-build` (exit status: 101)
  --- stdout
  cargo:rerun-if-changed=api/

  --- stderr
  api/../rocksdb/include/rocksdb/c.h:65:10: fatal error: 'stdarg.h' file not found
  thread 'main' panicked at 'unable to generate rocksdb bindings: ClangDiagnostic("api/../rocksdb/include/rocksdb/c.h:65:10: fatal error: 'stdarg.h' file not found\n")', /home/martin/.cargo/registry/src/github.com-1ecc6299db9ec823/oxrocksdb-sys-0.3.7/build.rs:28:10
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...
error: failed to compile `oxigraph_server v0.3.8`, intermediate artifacts can be found at `/tmp/cargo-installA1C1RJ`
```

I am on Debian 11.

**Comment by Tpt** [comment]
Hi! Thank you for trying Oxigraph. This looks like a C++ compile error on RocksDB (Oxigraph statically links RocksDB). The error message seems to state that the used C++ compiler (taken from the `CXX` env variable) does not know how to find the C standard library. Have you tweaked the `CC` and `CXX` env variables?

There is, for reference, [the documentation of the Rust wrapper on top of C/C++ compilers that is used during Oxigraph build](https://github.com/rust-lang/cc-rs)/

**Comment by not-my-profile** [comment]
Hi :) No I have not tweaked those environment variables.

**Comment by Tpt** [comment]
Thank you! Is the `build-essential` deb package installed? In the meantime you might just try [the last release pre built binary](https://github.com/oxigraph/oxigraph/releases/download/v0.3.8/oxigraph_server_v0.3.8_x86_64_linux_gnu). It should work with Bullseye.

**Comment by not-my-profile** [comment]
Yes I have `build-essential` installed. And yes thanks that binary does work.

**Comment by Tpt** [comment]
Thank you! It's weird. It seems to be related to a compiler configuration issue but I'm not sure how to fix it: https://stackoverflow.com/questions/20206290/fatal-error-stdarg-h-file-not-found

**Comment by adamreichold** [comment]
@not-my-profile I think it would be interesting to try building RocksDB itself on the system in question, following e.g. https://github.com/facebook/rocksdb/blob/main/INSTALL.md, to figure out if the incompatibility is with RocksDB's build system of with how the Rust wrapper drives it.

**Comment by not-my-profile** [comment]
Good idea! Yes running `make` or `make static_lib` in the RocksDB repo works flawlessly.

**Comment by Tpt** [comment]
Thank you! So, the problem seems related to the [`cc` crate](https://github.com/rust-lang/cc-rs).
